.travis.yml

```diff
@@ -20,6 +22,20 @@ matrix:
       osx_image: xcode7
       env: PYTHON=3.5 PYTHONWARNINGS=ignore
 
+    - os: windows
+      language: shell  # 'language: python' is an error on Travis CI Windows
+      before_install: choco install python2
```

Comment:
I'm not familiar with `choco`, but what do you think about just getting Miniconda from https://docs.conda.io/en/latest/miniconda.html and installing that?

Reply:
Actually, I think we should move this logic into https://github.com/ray-project/ray/blob/master/ci/travis/install-dependencies.sh, which is where we install Python for the other Travis builds.